includes/cub.h

```diff
@@ -126,4 +211,7 @@ int		count_appearances(char *str, char c);
 int		ft_split_len(char **split);
 int		is_number(char *str);
 
+/* WORLD MAP FOR TESTING */
```

Comment:
can we remove this comment if not relevant anymore?

Reply:
Done!